lib/rules/boolean-prop-naming.js

```diff
@@ -35,6 +35,10 @@ module.exports = {
           default: '^(is|has)[A-Z]([A-Za-z0-9]?)+',
           minLength: 1,
           type: 'string'
+        },
+        message: {
+          default: '',
```

Comment:
I don't think eslint pays attention to `default` here, fwiw.

Also, could we add `minLength: 1`? nobody should need to configure an empty string